etl/steps/export/multidim/education/latest/literacy.py

```diff
@@ -0,0 +1,352 @@
+                "view_config": view_config,
+                "view_metadata": view_metadata,
+            },
+            {
+                "dimension": "age_group",
+                "choice_new_slug": "age_side_by_side",
+                "choices": {"adult", "youth", "elderly"},
+                "view_config": view_config,
+                "view_metadata": view_metadata,
+            },
+        ],
+        params={
+            "title": lambda view: generate_title_by_gender_and_age(view),
+            "subtitle": lambda view: generate_subtitle_by_age_and_gender(view),
+        },
+    )
+
+
+def generate_title_by_gender_and_age(view):
+    """Generate title based on gender and age group."""
+    sex, age_group = view.dimensions["sex"], view.dimensions["age_group"]
+
+    # Get gender and age terms
+    gender_term = GENDERS.get(sex, {}).get("title", "")
+
+    if age_group == "age_side_by_side":
+        return "Literacy rates by age group"
+    elif sex == "sex_side_by_side":
+        age_config = AGE_GROUPS.get(age_group, {})
+        return f"Literacy rates among {age_config.get('title_term', age_group)}, by gender"
+    elif sex == "both":
+        age_config = AGE_GROUPS.get(age_group, {})
+        return f"Literacy rates among {age_config.get('title_term', age_group)}"
+    else:
+        # For specific genders, create better titles
+        age_config = AGE_GROUPS.get(age_group, {})
+        prefix = age_config.get("gender_prefix", "")
+        return f"Literacy rates among {prefix}{gender_term}"
+
+
+def _get_age_descriptions_by_gender(sex):
+    """Get age descriptions formatted for specific gender."""
+    if sex == "both":
+        return "adults aged 15 and above, young people (15–24), and older adults (65 and above)"
+    elif sex == "male":
+        return "men aged 15 and above, young men (15–24), and older men (65 and above)"
+    elif sex == "female":
+        return "women aged 15 and above, young women (15–24), and older women (65 and above)"
+    else:
+        return None
+
+
+def _get_gender_specific_subtitle(age_group, gender_term):
+    """Get gender-specific subtitle for specific age groups."""
+    age_config = AGE_GROUPS.get(age_group, {})
+    prefix = age_config.get("gender_prefix", "")
+    age_range = age_config.get("age_range", "")
+
+    return f"Share of {prefix}{gender_term} aged {age_range} who can read and write a short, simple sentence with understanding."
+
+
+def _get_sex_side_by_side_subtitle(age_group):
+    """Get subtitle for gender side-by-side comparisons."""
+    age_config = AGE_GROUPS.get(age_group, {})
+    prefix = age_config.get("gender_prefix", "")
+    age_range = age_config.get("age_range", "")
+
+    if prefix:
+        return f"Share of {prefix}women and men aged {age_range} who can read and write a short, simple sentence with understanding."
+    else:
+        return f"Share of women and men aged {age_range} who can read and write a short, simple sentence with understanding."
```

Comment:
I think you don't need the `if-else` here, just go with :


```suggestion
    return f"Share of {prefix}women and men aged {age_range} who can read and write a short, simple sentence with understanding."
```

If `prefix` evaluates to `False` (i.e. is equal to `""`), both texts will be equivalent.